exporters/sql_export/lc_for_jobs_db.py

```diff
@@ -0,0 +1,80 @@
+""" This program makes an actual state sqlite-database for use by
+jobs run under job-runner as opposed to the actual state used
+by customers. It is meant to be run just after the nightly imports
+to be used as a speed up in comparison with hitting MO's rest interface.
+"""
+import click
+import json
+import logging
+import pathlib
+
+
+from exporters.sql_export.sql_export import SqlExport
+from sqlalchemy import create_engine
```

Comment:
```suggestion
import json
import logging
import pathlib

import click
from sqlalchemy import create_engine

from exporters.sql_export.sql_export import SqlExport
```
`isort` er trist over imports her.